homeassistant/components/rainforest_eagle/sensor.py

```diff
@@ -46,6 +47,9 @@
         vol.Required(CONF_IP_ADDRESS): cv.string,
         vol.Required(CONF_CLOUD_ID): cv.string,
         vol.Required(CONF_INSTALL_CODE): cv.string,
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=list(SENSORS)): vol.All(
```

Comment:
Please create a constant for `list(SENSORS)` named `DEFAULT_MONITORED_CONDITIONS`